stockfish/models.py

```diff
@@ -728,6 +719,208 @@ def will_move_be_a_capture(self, move_value: str) -> Capture:
         else:
             return Stockfish.Capture.NO_CAPTURE
 
+    def convert_human_notation_to_sf_notation(self, move: str) -> str:
+        """
+        Args:
+            move:
+                A string which is the notation for a move, in a format that's used
+                by humans. E.g., stuff like Nf3, bxe5, etc.
+        Returns:
+            A string representation the notation for this move, in the form that
+            SF uses. E.g., Nf3 might become g1f3.
+        """
+
+        move = move.replace(" ", "").replace("-", "")
```

Comment:
may need to be more vigorous with replacements.
This was done to catch stuff like `o-o-o` or `B d x c6` more simply, but this allows strings such as `----o   -- --   oo ---   `, which is a bit silly.